src/Hl7.FhirPath/FhirPath/CompiledExpression.cs

```diff
@@ -31,6 +30,18 @@ public static bool Predicate(this CompiledExpression evaluator, ITypedElement in
                 return result.Value;
         }
 
+        //Empty is considered false
+        public static bool IsTrue(this CompiledExpression evaluator, ITypedElement input, EvaluationContext ctx)
+        {
+            var result = evaluator(input, ctx).BooleanEval();
+
+            if (result == null)
```

Comment:
I think we should get used to using `result is null` as the preferred syntax now.